adal/changelog.txt

```diff
@@ -1,3 +1,76 @@
+Version 4.0.0-preview
+==============
+This release includes:
```

Comment:
>his release includes: [](start = 1, length = 21)

IMO: both this and MSAL notes should reference the blog post via a link to the wiki?

Reply:
Agree. I will make the change accordingly.

---
In reply to: [213016360](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1230#discussion_r213016360) [](ancestors = 213016360)